### ISSUE TITLE
Missing Method in last update

### ISSUE THREAD

**Issue by McyD** [Tyrsis/SE-Community-Mod-API#44]
2015-01-29 18:04:11.929 - Thread: 1 -> Log Started
2015-01-29 18:04:11.929 - Thread: 1 -> Timezone (local - UTC): -5h
2015-01-29 18:04:11.930 - Thread: 1 -> App Version: 0.3.0.1
2015-01-29 18:04:11.930 - Thread: 1 -> Failed to find method '582447224E2B03FA4EAB3D6C2DDD48D9' in type '5F381EA9388E0A32A8C817841E192BE8.0F8EE1AD651CB822CB9635B463AE6CD5'

LightEnity Refection Validation Fails.

**Comment by Tyrsis** [comment]
Yes I left that one there for now, as I didn't have enough time to fix it (it was not a direct update).  The entire lighting synchronization class has changed, so I need to go through it when I get a chance.

**Comment by McyD** [comment]
Ah, Okay.
